src/CLI/CLI.csproj

```diff
@@ -55,6 +49,21 @@
     <ProjectReference Include="..\Automation\Automation.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This block is boilerplate that Visual Studio added when adding the resource file to the project